tests/rule_based_profiler/bobby_user_workflow_verbose_profiler_config.yml

```diff
@@ -2,12 +2,15 @@
 variables:
   # BatchRequest yielding two batches (January, 2019 and February, 2019 trip data)
   jan_feb_2019_trip_data_batch_request:
+    # We want to use the same triumvirate as in the user's batch_request.
```

Comment:
What do you mean by "user's batch_request?" Does that mean that we currently have two separate BatchRequests required for this profiler? Where does the other one come in?